docs/DeveloperGuide.md

```diff
@@ -2,8 +2,41 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
-  {:toc}
+### Table of Contents
+- [**Acknowledgements**](#--acknowledgements--)
```

Comment:
Although it is great that you have added a Table of Contents, I don't think this anchor link would work.
A great tip to have when setting up anchor links is to use the chrome inspector tool:
![image](https://user-images.githubusercontent.com/61146198/140596865-aa79a4e4-df9a-4dc6-bea9-8fc72eaba3e0.png)
You can just right click the header that you want an anchor link of, then find the `id` part of the html element.
In this case, the correct anchor link should be `(#acknowledgements)`.

You may also gain an intuition of how Github parses their anchor links:
![image](https://user-images.githubusercontent.com/61146198/140596966-ce2eec28-cb3a-4902-86e1-f1f8076f2827.png)
You can read further [here](https://gist.github.com/rachelhyman/b1f109155c9dafffe618)